client/package.json

```diff
@@ -71,12 +65,12 @@
     "globby": "^11.0.3",
     "happy-dom": "^2.19.0",
     "include-media": "^1.4.10",
-    "lerna": "^4.0.0",
     "nabs": "^0.3.0",
     "nyc": "^15.1.0",
     "postcss": "^8.2.15",
     "prettier": "^2.3.0",
     "prettier-plugin-svelte": "2.3.0",
+    "rollup": "^2.47.0",
```

Comment:
Added `rollup` to satisfy some peer dependency complaints - the only ones left are now because we're on the TypeScript beta.